RELEASES.md

```diff
@@ -380,3 +380,21 @@ The deprecated features are shown in the following table:
 | Built-in `aufs` snapshotter                                                      | containerd v1.5     | containerd v2.0            | Use `overlayfs` snapshotter       |
 | `cri-containerd-*.tar.gz` release bundles                                        | containerd v1.6     | containerd v2.0            | Use `containerd-*.tar.gz` bundles |
 | Pulling Schema 1 images (`application/vnd.docker.distribution.manifest.v1+json`) | containerd v1.7     | containerd v2.0            | Use Schema 2 or OCI images        |
+| CRI `v1alpha2`                                                                   | containerd v1.7     | containerd v2.0            | Use CRI `v1`                      |
+
+## Experimental features
+
+Experimental features are new features added to containerd which do not have the
+same stability guarantees as the rest of containerd. An effort is made to avoid
+breaking interfaces between versions, but changes to experimental features before
+being fully supported is possible. Users can still expect experimental features
+to be high quality and are encouraged to use new features to help them stabilize
+more quickly.
+
+| Component                  | Target release for full support |
+|----------------------------|---------------------------------|
+| Sandbox Service            | containerd v2.0                 |
+| Sandbox CRI Server         | containerd v2.0                 |
+| Transfer Service           | containerd v2.0                 |
+| NRI                        | containerd v2.0                 |
+| gRPC Shim                  | containerd v2.0                 |
```

Comment:
This seems experimental too
- https://github.com/containerd/containerd/pull/6899